test/integration/history_int_test.go

```diff
@@ -19,7 +19,13 @@ func (suite *HistoryIntegrationTestSuite) TestHistory_History() {
 
 	ts.LoginAsPersistentUser()
 	cp := ts.Spawn("history", "--namespace", "ActiveState-CLI/History")
-	cp.Expect(`added Platform`)
+	cp.Expect("Commit")
+	cp.Expect("Author")
+	cp.Expect("Date")
+	cp.Expect("Commit Message")
+	cp.Expect("+ autopip 1.6.0")
+	cp.Expect("- convertdate")
+	cp.Expect(`+ Platform`)
```

Comment:
Can we add a test here for seeing the version change? Since that's the main thing this story is addressing.